gef-ui/server/utils/helpers.js

```diff
@@ -123,7 +119,10 @@ const mapSummaryList = (data, itemsToShow) => {
         items: [
           ...(href ? [{
             href,
-            text: `${value ? 'Change' : 'Add'}`,
+            /* Clean-Deep removes any properties with Null value from an Object. Therefore if all
+            properties are Null, this leaves us with an Empty Object. isEmpty checks to see if the
+            Object is empty or not. */
+            text: `${!_isEmpty(cleanDeep(value)) ? 'Change' : 'Add'}`,
```

Comment:
probably worth putting into it's on function as it's duplicated, maybe something like

```
const isEmpty = (value) =>
  _isEmpty(cleanDeep(value)
```

Reply:
Done. Also added unit tests for it too